doc/Glossary/index.md

```diff
@@ -5,74 +5,198 @@ permalink: /:path/
-| Transport channel                                       | A Kaa client component responsible for communication with a Kaa server. Refer to Transports for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     |
-| U                                                       |                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| User verifier                                           | A server component that handles the user verification. Refer to User verifiers for more information.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
A business entity that uses a part or a whole [Kaa instance]({{root_url}}Glossary/#kaa-instance-kaa-deployment) to independently run [Kaa applications]({{root_url}}Glossary/#kaa-application).